README.md

```diff
@@ -90,6 +91,161 @@ This WordPress project is intended to provide:
   define( 'SCRIPT_DEBUG', true );
   ```
 
+## Automated Testing
```

Comment:
Let's move this section to its own wiki page in repo and link to that from here.